openassessment/assessment/models/base.py

```diff
@@ -828,3 +832,22 @@ def _check_all_criteria_assessed(cls, rubric_index, selected_criteria, criteria_
         if missing_criteria:
             msg = u"Missing selections for criteria: {missing}".format(missing=', '.join(missing_criteria))
             raise InvalidRubricSelection(msg)
+
+
+@python_2_unicode_compatible
```

Comment:
I am curious, what is this attribute fore?

Reply:
the decorator is to support python2 & python3 in a single codebase.  Basically, using it for QOL reasons